redis/alpine/label-updater.sh

```diff
@@ -0,0 +1,19 @@
+# Push some helpful vars into labels
+PODIP=`hostname -i`
+echo podIP $PODIP
+kubectl label --overwrite pod $HOSTNAME podIP="$PODIP"
```

Comment:
`kubectl` should not be configured with k8s `api-server` ? by default it will try to connect to `localhost:8080`.

Reply:
when run from inside a pod it will use the service account credentials for auth and will hit the kubernetes.local address.

Reply:
Ok @viglesiasce